.github/workflows/release-java.yml

```diff
@@ -0,0 +1,27 @@
+name: Release for Humbug Java library
+
+on:
+  push:
+    tags:
+      - 'java/v*'
+
+defaults:
+  run:
+    working-directory: java
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Publish package
+        run: mvn --batch-mode deploy
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_RELEASE_TOKEN }}
```

Comment:
use `secrets.GITHUB_TOKEN`. This is a special secret that gets generated dynamically for each GitHub Action workflow. More info on the `GITHUB_TOKEN` docs - https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
Done